### DISCUSSION TITLE
Tang nano HDMI video output

### DISCUSSION THREAD

**Discussion by somhi** [nand2mario/nestang#10] [Q&A]
Is it always required to use the frame buffer to be able to get HDMI video ouput ?

Galaga core video resolution is none of the defined ones for VIDEO_ID_CODE.

**Comment by nand2mario** [comment]
Probably go for 720p or 480p as your output resolution (the nano cannot reliably do 1080p). What I did with NES is to have a frame buffer of NES resolution (dual-port BRAM buffer). Then upscale the image to 720p on the fly.

Currently the frame buffer is BRAM. However it should also be possible to move it to SDRAM if it does not fit in BRAM. Then you need to utilize the SDRAM multi-bank ability.
